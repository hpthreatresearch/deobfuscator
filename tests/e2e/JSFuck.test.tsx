// Â© Copyright 2023 HP Development Company, L.P.
import { assertDeobfuscation } from "./assertDeobfuscation";

test(
  "obfuscator.io recipe",
  async () => {
    const deobfuscated = await assertDeobfuscation(
      "/Hello World/helloWorld.JSFuck.js",
      "JSFuck"
    );

    expect(deobfuscated.indexOf("Hello, World!") >= 0).toBe(true);
  },
  2 * 60000
);
