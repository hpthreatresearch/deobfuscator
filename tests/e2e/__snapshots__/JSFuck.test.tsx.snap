// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`obfuscator.io recipe 1`] = `
{
  "value": "(function () {  function hi() {    console.log("Hello, World!");  }  hi();})();",
}
`;
